packages/eds-core-react/src/components/Popover/PopoverHeader.tsx

```diff
@@ -0,0 +1,36 @@
+import { forwardRef, HTMLAttributes } from 'react'
+import styled, { css } from 'styled-components'
+import { Divider } from '../Divider'
+
+export type PopoverHeaderProps = HTMLAttributes<HTMLDivElement>
+
+const StyledPopoverHeader = styled.div`
+  display: grid;
+  grid-auto-flow: column;
+`
+
+const StyledDivider = styled(Divider)(({ theme }) => {
+  return css`
+    margin-left: -${theme.spacings.left};
```

Comment:
We want to avoid "hacks" with negative margins as it causes some unwanted consequences with different content. Maybe do the same refactor with spacings in `Popover` as we did with `Card`?

Reply:
I added `position: absolute` with `left`&`right` 0. It does work.
What do you think about it?

Reply:
I think it would be better to just fix this properly with doing the paddings the same way we did for `Card`. Its easier to maintain and more predicated for our users. Were you thinking you wanted a separate issue for that task?

Reply:
I see your point. 
Im not sure if i need a separate issue for it. The issue is a breaking change already. Would it be enough to make a clear description on the commits?
Also i found negative value in `Popover.Title`, i think it should be fixed to.

Reply:
Ok, you can do the changes in this PR as they are related to this refactor which as you say is already breaking. We should no longer need any of these negative margins if we fix the padding by moving them them to the sub-components :) 

What description we have on the commits doesn't matter in this case as they get squished on merge anyways. Suggest you just updated the description for this PR with the information instead :)

Reply:
Thanks, I will do that. 👌